CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/ResultRequestHandler.java

```diff
@@ -0,0 +1,13 @@
+package com.couchbase.CouchbaseLiteServ.server.RequestHandler;
```

Comment:
Do we need this class, as we don't have any test for them?

Reply:
no need to cover tests in API? @raghusarangapani?

Reply:
its ok to leave them in there as they are already written. Might be needed in the future.